R/utilities.R

```diff
@@ -2553,26 +2885,30 @@ get_gambl_colours = function(classification = "all",
 }
 
 
-#' Get full paths for bam files for a sample or patient.
+#' @title Get BAMs.
+#' 
+#' @description Get full paths for bam files for a sample or patient.
+#' 
+#' @details Returns a list with BAM paths for tumour, normal and mrna data.
+#' This function expects a sample ID (`sample`) or a patient ID (`patient`).
 #'
 #' @param sample Either provide sample_id or patient_id.
 #' @param patient Either provide sample_id or patient_id.
 #'
 #' @return A list that contains the genome_build and an igv-friendly build (igv_build), a list of bam file paths for tumour, normal and mrna data.
+#' 
+#' @import dplyr utils
```

Comment:
The utils is a default R package always available when R starts. Do we need to explicitly import it here? Also, couldn't see where it is used below

Reply:
I'll go over all the code that currently imports `utils` based on the information you provided. In most cases, the functions called from this package are `head`, `tail`, `read_table` and `write_table`.